interfaces/builtin/bluez.go

```diff
@@ -196,13 +205,38 @@ func (iface *BluezInterface) PermanentPlugSnippet(plug *interfaces.Plug, securit
 
 func (iface *BluezInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
 	switch securitySystem {
-	case interfaces.SecurityDBus:
-		return bluezConnectedPlugDBus, nil
 	case interfaces.SecurityAppArmor:
-		return bluezConnectedPlugAppArmor, nil
+		old := []byte("@SLOT_SECURITY_TAGS@")
+		var new []byte
+		switch {
+		case len(slot.Apps) == 1:
+			for appName := range slot.Apps {
```

Comment:
I see `len(slot.Apps) == 1` and then a `range slot.Apps`. That loop seems to be not required if we know the size is just one? Or am I missing something?